iree/hal/vulkan/api.h

```diff
@@ -55,6 +49,19 @@ typedef struct {
   iree_hal_vulkan_extensibility_options_t extensibility_options;
 } iree_hal_vulkan_driver_options_t;
 
+// A list of queues within a specific queue family on a VkDevice.
+typedef struct {
+  // The index of a particular queue family on a VkPhysicalDevice, as described
+  // by vkGetPhysicalDeviceQueueFamilyProperties.
+  uint32_t queue_family_index;
+
+  // A list of queue indices within the queue family at |queue_family_index|.
+  uint32_t* queue_indices;
+
+  // The number of elements in |queue_indices|.
+  uint32_t queue_indices_count;
+} iree_hal_vulkan_queues_info_t;
```

Comment:
"queues info" pluralization is a bit odd as then when looking at values it's confusing as to whether a pointer is a list of the structs or just one of the structs. just iree_hal_vulkan_queue_set_t would let you avoid the issue.

Reply:
Renamed to `iree_hal_vulkan_queue_set_t`